public/vendor/horizon/app-dark.css

```diff
@@ -0,0 +1,8 @@
+.vjs__tree .vjs__tree__content{border-left:1px dotted hsla(0,0%,80%,.28)!important}.vjs__tree .vjs__tree__node{cursor:pointer}.vjs__tree .vjs__tree__node:hover{color:#20a0ff}.vjs__tree .vjs-checkbox{position:absolute;left:-30px}.vjs__tree .vjs__value__boolean,.vjs__tree .vjs__value__null,.vjs__tree .vjs__value__number{color:#a291f5!important}.vjs__tree .vjs__value__string{color:#dacb4d!important}.hljs-addition,.hljs-keyword,.hljs-selector-tag{color:#8bd72f}.hljs-doctag,.hljs-meta .hljs-meta-string,.hljs-regexp,.hljs-string{color:#dacb4d}.hljs-literal,.hljs-number{color:#a291f5!important}
```

Comment:
We don't need to hardcore this files because they are in the installation process